src/Kiota.Builder/Configuration/SearchConfiguration.cs

```diff
@@ -4,5 +4,12 @@ namespace Kiota.Builder.Configuration;
 
 public class SearchConfiguration : SearchConfigurationBase {
     public Uri APIsGuruListUrl { get; set; } = new ("https://raw.githubusercontent.com/APIs-guru/openapi-directory/gh-pages/v2/list.json");
-    public Uri GitHubBlockListUrl { get; set; } = new ("https://raw.githubusercontent.com/microsoft/kiota/main/resources/index-block-list.yml");
+    public GitHubConfiguration GitHub { get; set; } = new();
+}
+
+public class GitHubConfiguration {
+    public string AppId { get; set; } = "Iv1.9ed2bcb878c90617";
```

Comment:
I believe these values are loaded from the configuration. Any specific reason we need to have defaults hardcoded here?

Reply:
it's a safety net just in case the configuration is not available. We don't own the deployment of Kiota itself, people could decide to drop the configuration for whatever reason.